src/scenes/levelend-scene/levelend-scene.ts

```diff
@@ -80,19 +97,19 @@ export class LevelEndScene {
       if (isDocumentVisible()) {
         this.audioPlayer.playAudio(AUDIO_LEVEL_LOSE);
       }
-      // this.monster.changeToSpitAnimation(); //Commenting to handle interactive animation
+      this.riveMonster.play(RiveMonsterComponent.Animations.EAT_DISGUST);
     } else {
       if (isDocumentVisible()) {
         this.audioPlayer.playAudio(AUDIO_LEVEL_WIN);
         this.audioPlayer.playAudio(AUDIO_INTRO);
       }
-      // this.monster.changeToEatAnimation();  //Commenting to handle interactive animation
+      this.riveMonster.play(RiveMonsterComponent.Animations.EAT_HAPPY);
```

Comment:
Would be great if you can write a unit test for this line.

Reply:
added unit test cases